passwd.master

```diff
@@ -42,3 +42,4 @@ g-s-helper:*:122:129:GNOME Software helper system user,,,:/var/lib/g-s-helper/:/
 ndn-user:*:123:130::/var/lib/ndn:/bin/false
 systemd-network:*:124:131:systemd Network Management,,,:/run/systemd/netif:/bin/false
 systemd-resolve:*:125:132:systemd Resolver,,,:/run/systemd/resolve:/bin/false
+companion-app-helper:*:126:132:Companion Application helper system user,,,:/var/lib/companion-app-helper/:/usr/sbin/nologin
```

Comment:
Does this directory actually exist? Can we leave it blank?

Reply:
Also, the group should be `133`

Reply:
Ah, yikes - thanks for catching, I thought I had incremented that.

As for the directory ... I don't *think* it has to exist, at least looking at some of the other users which have a home directory that does not exist on my system. However the `metrics` user also exists to spawn a system service has it has no homedir:

    metrics:*:113:120:Endless Metrics Daemon::/usr/sbin/nologin

So maybe we don't need one in this case.

Reply:
Okay, bumped the UID.

It looks like this directory *does* need to be created, otherwise flatpak will attempt to create `/.var`. I'm not sure how that's done.